Testing/Cxx/CMakeLists.txt

```
@@ -0,0 +1,46 @@
+# AdaptiveCpp testing
+if(EXTENSION_BUILDS_ADAPTIVE_CPP AND NOT WIN32)
+
+  # Find required packages
+  find_package(AdaptiveCpp CONFIG REQUIRED HINTS ${AdaptiveCpp_DIR})
+  find_package(CLI11 CONFIG REQUIRED HINTS ${CLI11_DIR})
+
+  # Create the test executable
+  add_executable(basic_SYCL_test basic_SYCL_test.cpp)
+
+  # Link libraries
+  target_link_libraries(basic_SYCL_test PRIVATE CLI11::CLI11)
+
+  # Add SYCL support
+  add_sycl_to_target(TARGET basic_SYCL_test)
```

```suggestion
  add_sycl_to_target(TARGET basic_SYCL_test SOURCES basic_SYCL_test.cpp)
```
in this case it's redundant, but is useful if you have a library where you have some sources with SYCLS and others without it. For example:
```
     # Compile specific sources for SYCL and build target for SYCL
     add_executable(target_proj A.cpp B.cpp offload1.cpp offload2.cpp)
     add_sycl_to_target(TARGET target_proj SOURCES offload1.cpp offload2.cpp)
```